langchain-python/indexes/langchain_test_vectorstores_from_texts.json

```diff
@@ -0,0 +1,18 @@
+{
+  "fields": [
+    {
+      "numDimensions": 5,
```

Comment:
I see some tests now have 5 dimensions whilst others have 1536. Any hesitation to making them all use 1536?

Reply:
The ones that use 5 are that way because we're not testing the quality of the search. If not, anything to make it faster.